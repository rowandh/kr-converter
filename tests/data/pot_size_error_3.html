
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-3-29522298</p></td>
                                <td height="26" ><p align="center">&nbsp;2025-02-05 오후 2:20:29</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>Wnrdjqhwk (A 5 투 페어)</b></p></td>
<!--								<td ><p align="center">&nbsp;243470</p></td>   -->
								<td ><p align="center">&nbsp;243,470</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dgjdkdk</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>7 <font color='red'>◆</font>6 [7 탑]  (다이) 
<br>* 시작 : [StageNo:29522298] [Credit:741,372원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:300]
<br>* NICKNAME:[dgjdkdk]
<br>* 앤티: -2,000원(739,372원)
<br>* 홀 카드딜: <font color='red'>◆</font>7(19) <font color='red'>◆</font>6(18) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(<font color='red'>◆</font>7 <font color='red'>◆</font>6)]
<br>* 베팅: 다이 [0](739,372원) - 베팅순서: [0][2]
[634ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>◆</font>6) C (♠10♠A<font color='red'>♥</font>J)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>◆</font>6) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>◆</font>6) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 종료: WinMoney[0원] Credit[739,372원]
<br>* 결과: 패배 [족보:7 탑] [카드:<font color='red'>◆</font>7 <font color='red'>◆</font>6] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">739,372</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ynfmngaszn</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠7 ♣2 [7 탑]  (다이) 
<br>* 시작 : [StageNo:29522298] [Credit:400,000원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:200]
<br>* NICKNAME:[ynfmngaszn]
<br>* 앤티: -2,000원(398,000원)
<br>* 홀 카드딜: ♠7(6) ♣2(40) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(♠7 ♣2)]
<br>* 베팅: 다이 [0](398,000원) - 베팅순서: [0][6]
[3601ms]
<br>* 커뮤니티 카드 딜: H(♠7♣2) C (♠10♠A<font color='red'>♥</font>J)
<br>* 커뮤니티 카드 딜: H(♠7♣2) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 커뮤니티 카드 딜: H(♠7♣2) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 종료: WinMoney[0원] Credit[398,000원]
<br>* 결과: 패배 [족보:7 탑] [카드:♠7 ♣2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">398,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>jnfvksdhc</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣8 ♣4 [8 탑]  (다이) 
<br>* 시작 : [StageNo:29522298] [Credit:589,831원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:300]
<br>* NICKNAME:[jnfvksdhc]
<br>* 앤티: -2,000원(587,831원)
<br>* 홀 카드딜: ♣8(46) ♣4(42) [8 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:8 탑(♣8 ♣4)]
<br>* 베팅: 다이 [0](587,831원) - 베팅순서: [0][4]
[1096ms]
<br>* 커뮤니티 카드 딜: H(♣8♣4) C (♠10♠A<font color='red'>♥</font>J)
<br>* 커뮤니티 카드 딜: H(♣8♣4) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 커뮤니티 카드 딜: H(♣8♣4) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 종료: WinMoney[0원] Credit[587,831원]
<br>* 결과: 패배 [족보:8 탑] [카드:♣8 ♣4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">587,831</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ringgg27</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>A ♣7 [A 탑]  (다이) 
<br>* 시작 : [StageNo:29522298] [Credit:435,529원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:100]
<br>* NICKNAME:[ringgg27]
<br>* 앤티: -2,000원(433,529원)
<br>* 홀 카드딜: <font color='red'>◆</font>A(13) ♣7(45) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(<font color='red'>◆</font>A ♣7)]
<br>* 베팅: 다이 [0](433,529원) - 베팅순서: [0][1]
[1106ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A♣7) C (♠10♠A<font color='red'>♥</font>J)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A♣7) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A♣7) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 종료: WinMoney[0원] Credit[433,529원]
<br>* 결과: 패배 [족보:A 탑] [카드:<font color='red'>◆</font>A ♣7] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">433,529</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>oguiytreg</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠Q <font color='red'>♥</font>8 [Q 탑]  (다이) (Entry Fee)
<br>* 시작 : [StageNo:29522298] [Credit:600,000원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:300]
<br>* NICKNAME:[oguiytreg]
<br>* 앤티: -2,000원(598,000원)
<br>* 홀 카드딜: ♠Q(11) <font color='red'>♥</font>8(33) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(♠Q <font color='red'>♥</font>8)]
<br>* 신규 참가비 [금액:2,000원] [Credit:596,000원]
<br>* 베팅: 체크 -0원(596,000원) - 베팅순서: [0][3]
[200ms]
<br>* 베팅: 다이 [0](596,000원) - 베팅순서: [0][10]
[2936ms]
<br>* 커뮤니티 카드 딜: H(♠Q<font color='red'>♥</font>8) C (♠10♠A<font color='red'>♥</font>J)
<br>* 커뮤니티 카드 딜: H(♠Q<font color='red'>♥</font>8) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 커뮤니티 카드 딜: H(♠Q<font color='red'>♥</font>8) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 종료: WinMoney[0원] Credit[596,000원]
<br>* 결과: 패배 [족보:Q 탑] [카드:♠Q <font color='red'>♥</font>8] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-4,000</font></p></td>
											<td style="word-break:break-all"><p align="right">596,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>gfdgfjhtj</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣10 <font color='red'>◆</font>9 ♠10 ♠A <font color='red'>♥</font>J ♣5 ♣6 [10 원 페어]  
<br>* 시작 : [StageNo:29522298] [Credit:1,152,648원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:150]
<br>* NICKNAME:[gfdgfjhtj]
<br>* 앤티: -2,000원(1,150,648원)
<br>* 홀 카드딜: ♣10(48) <font color='red'>◆</font>9(21) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(♣10 <font color='red'>◆</font>9)]
<br>* 베팅: [레이즈] (16,000원) Credit(1,134,648원) - 베팅순서: [0][7]
[1949ms]
<br>* 베팅: [하프] (160,064원) Credit(974,584원) - 베팅순서: [0][11]
[3586ms]
<br>* 커뮤니티 카드 딜: H(♣10<font color='red'>◆</font>9) C (♠10♠A<font color='red'>♥</font>J)
<br>* 턴 시작: [플랍(1)] [족보:10 원 페어(♠10 ♣10 ♠A <font color='red'>♥</font>J <font color='red'>◆</font>9)]
<br>* 베팅: 체크 -0원(974,584원) - 베팅순서: [1][2]
[2179ms]
<br>* 커뮤니티 카드 딜: H(♣10<font color='red'>◆</font>9) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 턴 시작: [턴(2)] [족보:10 원 페어(♠10 ♣10 ♠A <font color='red'>♥</font>J <font color='red'>◆</font>9)]
<br>* 베팅: 체크 -0원(974,584원) - 베팅순서: [2][2]
[2067ms]
<br>* 커뮤니티 카드 딜: H(♣10<font color='red'>◆</font>9) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 턴 시작: [리버(3)] [족보:10 원 페어(♠10 ♣10 ♠A <font color='red'>♥</font>J <font color='red'>◆</font>9)]
<br>* 베팅: 체크 -0원(974,584원) - 베팅순서: [3][2]
[1416ms]
<br>* 종료: WinMoney[0원] Credit[974,584원]
<br>* 결과: 패배 [족보:10 원 페어] [카드:♠10 ♣10 ♠A <font color='red'>♥</font>J <font color='red'>◆</font>9]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-178,064</font></p></td>
											<td style="word-break:break-all"><p align="right">974,584</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Wnrdjqhwk</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣A <font color='red'>♥</font>5 ♠10 ♠A <font color='red'>♥</font>J ♣5 ♣6 [A 5 투 페어]  (Big Blind)
<br>* 시작 : [StageNo:29522298] [Credit:81,032원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:100]
<br>* NICKNAME:[wnrdjqhwk]
<br>* 앤티: -2,000원(79,032원)
<br>* 홀 카드딜: ♣A(39) <font color='red'>♥</font>5(30) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(♣A <font color='red'>♥</font>5)]
<br>* 베팅: [블라인드:BIG] [금액:2,000원] [Creadit:77,032원]
<br>* 베팅: [올인] (77,032원) Credit(0원) - 베팅순서: [0][9]
[1282ms]
<br>* 올인 플레이어 턴: Credit(0원)
<br>* 커뮤니티 카드 딜: H(♣A<font color='red'>♥</font>5) C (♠10♠A<font color='red'>♥</font>J)
<br>* 턴 시작: [플랍(1)] [족보:A 원 페어(♠A ♣A <font color='red'>♥</font>J ♠10 <font color='red'>♥</font>5)]
<br>* 올인 플레이어 턴: Credit(0원)
<br>* 커뮤니티 카드 딜: H(♣A<font color='red'>♥</font>5) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 턴 시작: [턴(2)] [족보:A 5 투 페어(♠A ♣A ♣5 <font color='red'>♥</font>5 <font color='red'>♥</font>J)]
<br>* 올인 플레이어 턴: Credit(0원)
<br>* 커뮤니티 카드 딜: H(♣A<font color='red'>♥</font>5) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 턴 시작: [리버(3)] [족보:A 5 투 페어(♠A ♣A ♣5 <font color='red'>♥</font>5 <font color='red'>♥</font>J)]
<br>* 올인 플레이어 턴: Credit(0원)
<br>* 종료: WinMoney[243,470원] Credit[243,470원]
<br>* 결과: 승리 [족보:A 5 투 페어] [카드:♠A ♣A ♣5 <font color='red'>♥</font>5 <font color='red'>♥</font>J]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>162,438</font></p></td>
											<td style="word-break:break-all"><p align="right">243,470</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>pihasdtre</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>4 ♠3 [4 탑]  (다이) 
<br>* 시작 : [StageNo:29522298] [Credit:120,774원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:150]
<br>* NICKNAME:[pihasdtre]
<br>* 앤티: -2,000원(118,774원)
<br>* 홀 카드딜: <font color='red'>◆</font>4(16) ♠3(2) [4 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:4 탑(<font color='red'>◆</font>4 ♠3)]
<br>* 베팅: 다이 [0](118,774원) - 베팅순서: [0][5]
[168ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>4♠3) C (♠10♠A<font color='red'>♥</font>J)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>4♠3) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>4♠3) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 종료: WinMoney[0원] Credit[118,774원]
<br>* 결과: 패배 [족보:4 탑] [카드:<font color='red'>◆</font>4 ♠3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">118,774</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>xzcvzswse</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>K ♠K ♠10 ♠A <font color='red'>♥</font>J ♣5 ♣6 [K 원 페어]  (Small Blind)
<br>* 시작 : [StageNo:29522298] [Credit:740,792원] [SB:2,000원] [BB:2,000원] [MBI:200,000원] [CBIR:150]
<br>* NICKNAME:[xzcvzswse]
<br>* 앤티: -2,000원(738,792원)
<br>* 홀 카드딜: <font color='red'>♥</font>K(38) ♠K(12) [K 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:K 원 페어(<font color='red'>♥</font>K ♠K)]
<br>* 베팅: [블라인드:SMALL] [금액:2,000원] [Creadit:736,792원]
<br>* 베팅: 콜 -14,000원(722,792원) - 베팅순서: [0][8]
[3333ms]
<br>* 베팅: 콜 -160,064원(562,728원) - 베팅순서: [0][12]
[2834ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K♠K) C (♠10♠A<font color='red'>♥</font>J)
<br>* 턴 시작: [플랍(1)] [족보:K 원 페어(<font color='red'>♥</font>K ♠K ♠A <font color='red'>♥</font>J ♠10)]
<br>* 베팅: 체크 -0원(562,728원) - 베팅순서: [1][1]
[2669ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K♠K) C (♠10♠A<font color='red'>♥</font>J) (♣5)
<br>* 턴 시작: [턴(2)] [족보:K 원 페어(<font color='red'>♥</font>K ♠K ♠A <font color='red'>♥</font>J ♠10)]
<br>* 베팅: 체크 -0원(562,728원) - 베팅순서: [2][1]
[4026ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K♠K) C (♠10♠A<font color='red'>♥</font>J) (♣5) (♣6)
<br>* 턴 시작: [리버(3)] [족보:K 원 페어(<font color='red'>♥</font>K ♠K ♠A <font color='red'>♥</font>J ♠10)]
<br>* 베팅: 체크 -0원(562,728원) - 베팅순서: [3][1]
[2905ms]
<br>* 종료: WinMoney[183,779원] Credit[746,507원]
<br>* 결과: 패배 [족보:K 원 페어] [카드:<font color='red'>♥</font>K ♠K ♠A <font color='red'>♥</font>J ♠10]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>5,715</font></p></td>
											<td style="word-break:break-all"><p align="right">746,507</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-3-29522298">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


