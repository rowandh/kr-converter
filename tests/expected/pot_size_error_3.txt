PokerStars Hand #15329522298:  Hold'em No Limit ($2000/$2000) - 2025/02/05 14:20:29 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: ringgg27 ($435529 in chips)
Seat 2: dgjdkdk ($741372 in chips)
Seat 3: oguiytreg ($600000 in chips)
Seat 4: jnfvksdhc ($589831 in chips)
Seat 5: pihasdtre ($120774 in chips)
Seat 6: ynfmngaszn ($400000 in chips)
Seat 7: gfdgfjhtj ($1152648 in chips)
Seat 8: xzcvzswse ($740792 in chips)
Seat 9: Wnrdjqhwk ($81032 in chips)
ringgg27: posts the ante $2000
dgjdkdk: posts the ante $2000
oguiytreg: posts the ante $2000
jnfvksdhc: posts the ante $2000
pihasdtre: posts the ante $2000
ynfmngaszn: posts the ante $2000
gfdgfjhtj: posts the ante $2000
xzcvzswse: posts the ante $2000
Wnrdjqhwk: posts the ante $2000
xzcvzswse: posts small blind $2000
Wnrdjqhwk: posts big blind $2000
oguiytreg: posts big blind $2000
*** HOLE CARDS ***
ringgg27: folds
dgjdkdk: folds
oguiytreg: checks
jnfvksdhc: folds
pihasdtre: folds
ynfmngaszn: folds
gfdgfjhtj: raises $14000 to $16000
xzcvzswse: calls $14000
Wnrdjqhwk: raises $63032 to $79032 and is all-in
oguiytreg: folds
gfdgfjhtj: raises $97032 to $176064
xzcvzswse: calls $160064
*** FLOP *** [Ts As Jh]
xzcvzswse: checks
gfdgfjhtj: checks
*** TURN *** [Ts As Jh] [5c]
xzcvzswse: checks
gfdgfjhtj: checks
*** RIVER *** [Ts As Jh] [5c] [6c]
xzcvzswse: checks
gfdgfjhtj: checks
Wnrdjqhwk collected $243470 from main pot
xzcvzswse collected $183779 from side pot
*** SUMMARY ***
Total pot $267381 | Rake $23911
Board [Ts As Jh 5c 6c]
Seat 1: ringgg27 mucked [Ad 7c]
Seat 2: dgjdkdk mucked [7d 6d]
Seat 3: oguiytreg mucked [Qs 8h]
Seat 4: jnfvksdhc mucked [8c 4c]
Seat 5: pihasdtre mucked [4d 3s]
Seat 6: ynfmngaszn mucked [7s 2c]
Seat 7: gfdgfjhtj showed [Tc 9d] and lost
Seat 8: xzcvzswse showed [Kh Ks] and won ($183779)
Seat 9: Wnrdjqhwk showed [Ac 5h] and won ($243470)